cardano-db-sync/src/Cardano/DbSync/Era/Shelley/Offline/Http.hs

```diff
@@ -31,6 +31,7 @@ import qualified Data.Text.Encoding as Text
 import Network.HTTP.Client (HttpException (..))
 import qualified Network.HTTP.Client as Http
 import qualified Network.HTTP.Types as Http
+import qualified GHC.Show as GHCS
```

Comment:
Is this used?

Reply:
this is weird when removing it gave me an error on the `show` inside the show instance. I removed the import and used `import GHC.Show (show)`